index.bs

```diff
@@ -378,37 +340,63 @@ When requesting the `webid` scope, the user's WebID MUST be present in the ID To
             "webid": "https://janedoe.com/web#id",
             "iss": "https://idp.example.com",
             "sub": "janedoe",
-            "aud": "https://client.example.com/web#id",
-            "nonce": "n-0S6_WzA2Mj",
-            "exp": 1311281970,
+            "aud": ["https://client.example.com/web#id", "solid"],
+            "azp": "https://client.example.com/web#id",
             "iat": 1311280970,
+            "exp": 1311281970,
+            "cnf":{
+              "jkt":"0ZcOCORZNYy-DWpqq30jZyJGHTN0d2HglBV3uiguA4I"
+            }
         }
     </pre>
 </div>
 
 # Resource Access # {#resource}
 
+## Authorization Server Discovery ## {#authorization-server-discovery}
+
+When client performs unauthenticated request to protected resource,
+Resource Server MUST respond with HTTP <code>401</code> status code,
+as well as <code>WWW-Authenticate</code> HTTP header including <code>UMA</code>
+as authentication scheme, with the <code>issuer</code> URI from the
+authorization server's discovery document in <code>as_uri</code> parameter
+and permission ticket in  <code>ticket</code> parameter.[[!UMA]]
+
+Given <code>as_uri</code> client can discover Authorization Server token endponint via
+Authorization Server Metadata available at <code>./well-known/uma2-configuration</code>.
+As specified in [[!UMA]]
+
+
+## Obtaining Access Token ## {#obtaining-access-token}
+
+Client can obtain Access Token from Authorization Server token endpoint using
+<code>urn:ietf:params:oauth:grant-type:uma-ticket</code> grant [[!UMA]].
```

Comment:
As mentioned separately, UMA should only be an example for managing token exchange rather than being a normative requirement for Solid-OIDC